lib/commands/run.ts

```diff
@@ -32,9 +32,9 @@ export class RunCommandBase implements ICommand {
 			this.platform = this.$devicePlatformsConstants.Android;
 		}
 
-		const availablePlatforms = this.platform ? [this.platform] : this.$platformsData.availablePlatforms;
-		for (let platform of availablePlatforms) {
-			const platformData = this.$platformsData.getPlatformData(platform, this.$projectData);
+		const availablePlatforms = this.platform ? {[this.platform]: this.platform} : this.$platformsData.availablePlatforms;
```

Comment:
The main reasoning for leaving it an object is that availablePlatforms is widely used inside the code base and the change will have bigger impact for little value.